<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FDK AAC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: white;
	font-family: &quot;Segoe UI&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="base_frame">
     <property name="styleSheet">
      <string notr="true">QFrame#base_frame {
	background-color: #333333;
	border-radius: 10px;

}

QComboBox QAbstractItemView {
	color: black;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="title_bar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color: #f77f00;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: white;
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;

}</string>
           </property>
           <property name="text">
            <string>FDK AAC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_title_bar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>677</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="main_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QFrame" name="bitrate_mode_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="3" column="0">
             <widget class="QRadioButton" name="cbr_radio">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton::indicator {
	background-color: #333333;
	border-radius: 6px;
	border: 1px solid #7f7f7f;
	width: 12px;
	height: 12px;
}

QRadioButton::indicator:hover {
	border: 1px solid #f77f00;
}

QRadioButton::indicator:pressed {
	border: 1px solid #f77f00;
	background-color: #222222;
}

QRadioButton::indicator:checked {
	image: url(:/icons/icons/check_white_18dp.svg)
}
</string>
              </property>
              <property name="text">
               <string>CBR</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QRadioButton" name="vbr_radio">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton::indicator {
	background-color: #333333;
	border-radius: 6px;
	border: 1px solid #7f7f7f;
	width: 12px;
	height: 12px;
}

QRadioButton::indicator:hover {
	border: 1px solid #f77f00;
}

QRadioButton::indicator:pressed {
	border: 1px solid #f77f00;
	background-color: #222222;
}

QRadioButton::indicator:checked {
	image: url(:/icons/icons/check_white_18dp.svg)
}
</string>
              </property>
              <property name="text">
               <string>VBR</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="cbr_input">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;

}

QLineEdit:hover {
	border: 1px solid #f77f00;
}

QLineEdit:focus {
	border: 1px solid #f77f00;
}

QLineEdit:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
              </property>
              <property name="text">
               <string>128k</string>
              </property>
              <property name="placeholderText">
               <string>128k</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="bitrate_title">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>15</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight: bold

}</string>
              </property>
              <property name="text">
               <string>Bitrate mode</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="vbr_spinbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QSpinBox {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;
	padding-left: 4px;

}

QSpinBox:hover {
	border: 1px solid #f77f00;
}

QSpinBox:focus {
	border: 1px solid #f77f00;
}

QSpinBox:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);
}

</string>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="confirm_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 7px;
	border: 1px solid #7f7f7f;
	padding: 0px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid rgb(33, 213, 93);
}

QPushButton:pressed {
	background-color: rgb(33, 213, 93);
}



</string>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/icons/icons/check_white_18dp.svg</normaloff>:/icons/icons/check_white_18dp.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbr_input</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbr_spinbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vbr_spinbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vbr_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbr_input</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
